ooni/tests/mocks.py

```diff
@@ -242,3 +242,19 @@ def isSupported(self):
 
     def isReachable(self):
         return defer.succeed(True)
+
+
+class MockDnsResponsePayload(object):
+    def dottedQuad(self):
+        return '127.0.0.1'
+
+
+class MockDnsResponse(object):
+    payload = MockDnsResponsePayload()
+
+
+def mock_dns_lookup_address():
+    response = (
+        [MockDnsResponse()],
+    )
+    yield response
```

Comment:
But the code where this is called it's yielding and defer.inlineCallbacks decorated. I've tried that but IIRC it breaks with not handling StopIteration.